anvio/drivers/prodigal.py

```diff
@@ -21,11 +26,108 @@
 __maintainer__ = "A. Murat Eren"
 __email__ = "a.murat.eren@gmail.com"
 
-
 run = terminal.Run()
 progress = terminal.Progress()
 pp = terminal.pretty_print
 
+# How many times to retry the gene calling.
+NUM_RETRIES = 3
```

Comment:
I am not sure why is this necessary :) In the function where this variable is used the description says "Sometimes commands can fail for weird reasons so retrying often fixes things", but it should never happen. If retrying is solving something, there must be somewhere else where there is a problem (i.e., race conditions, missing file.close() calls, etc) and we should find them. I think this variable complicates things unnecessarily.

Reply:
I think that's a fair point....I will remove the "retrying" code from the command runner.